packages/@lwc/engine/scripts/rollup/rollup.config.js

```diff
@@ -12,6 +12,9 @@ const { version } = require('../../package.json');
 const entry = path.resolve(__dirname, '../../src/framework/main.ts');
 const targetDirectory = path.resolve(__dirname, '../../dist/');
 
+const rollupLwcFeatures = require('@lwc/features/src/rollup-plugin');
+const featureFlags = require('@lwc/features');
```

Comment:
Couldn't the `featureFlags` parameter that you pass to the transformer be optional? This way you don't need to import the default feature flags to pass it again to the transformer.